docs/README.md

```diff
@@ -78,8 +78,126 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
 ## Multicluster
 tbd
 
-## Examples
-tbd
+## Addons
+
+Addons are managed separately from the Sail-operator. You can follow the [istio documentation](https://istio.io/latest/docs/ops/integrations/) for how to install addons. Below is an example of how to install some addons for Istio.
+
+The sample will deploy:
+
+- Prometheus
+- Jaeger
+- Kiali
+- Bookinfo demo app
+
+*Prerequisites*
+
+- Sail operator installed.
+- Control Plane installed with Sail.
+
+### Deploy Prometheus and Jaeger addons
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/prometheus.yaml
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/jaeger.yaml
+```
+
+### Deploy Kiali addon
+
+Install the kiali operator.
+
+You can install the kiali operator through OLM if running on Openshift, otherwise you can use helm:
+
+```sh
+helm install --namespace kiali-operator --create-namespace kiali-operator kiali/kiali-operator
+```
+
+Find out the revision name of your Istio instance. In our case it is `test`.
+    
+```bash
+$ kubectl get istiorevisions.operator.istio.io 
+NAME   READY   STATUS    IN USE   VERSION   AGE
+test True    Healthy   True     v1.21.0   119m
+```
+
+Create a Kiali resource and point it to your Istio instance. Make sure to replace `test` with your revision name in the fields `config_map_name`, `istio_sidecar_injector_config_map_name`, `istiod_deployment_name` and `url_service_version`.
```

Comment:
Should we create the ClusterRoleBinding as documented here - https://github.com/istio-ecosystem/sail-operator/tree/main/docs#integrating-with-kiali

Reply:
No that's only needed when you are using prometheus from openshift cluster monitoring. With the addon prometheus that this example uses, kiali doesn't need special permissions to access it.